README.md

```diff
@@ -58,6 +60,8 @@ TSDX comes with the "battery-pack included" and is part of a complete TypeScript
 - Jest test runner setup with sensible defaults via `tsdx test`
 - Zero-config, single dependency
 
+[🔝](#toc)<!-- Link generated with jump2header -->
```

Comment:
```suggestion
[🔝 Back to Top 🔝](#toc)<!-- Link generated with jump2header -->
```

Symbols are not the most accessible and this could be missed easily, so should add some text to each of these.